bcdaworker/queueing/manager/que.go

```diff
@@ -113,37 +114,44 @@ func (q *queue) processJob(job *que.Job) error {
 		return nil
 	}
 
+	ctxFields := logrus.Fields{"job_id": jobArgs.ID, "aco_id": jobArgs.ACOID}
+
+	ctx = workerlog.WithLogFields(ctx, ctxFields)
+
 	exportJob, err := q.worker.ValidateJob(ctx, jobArgs)
 	if goerrors.Is(err, worker.ErrParentJobCancelled) {
 		// ACK the job because we do not need to work on queue jobs associated with a cancelled parent job
-		q.log.Warnf("queJob %d associated with a cancelled parent Job %d. Removing queuejob from que.", job.ID, jobArgs.ID)
+		q.log.WithFields(ctxFields).Warnf("Removing queuejob from que; parent job %d cancelled.", jobArgs.ID)
 		return nil
 	} else if goerrors.Is(err, worker.ErrNoBasePathSet) {
 		// Data is corrupted, we cannot work on this job.
-		q.log.Warnf("Job %d does not contain valid base path. Removing queuejob from que.", jobArgs.ID)
+		q.log.WithFields(ctxFields).Warnf("Job does not contain valid base path; removing queuejob from que.")
 		return nil
 	} else if goerrors.Is(err, worker.ErrParentJobNotFound) {
 		// Based on the current backoff delay (j.ErrorCount^4 + 3 seconds), this should've given
 		// us plenty of headroom to ensure that the parent job will never be found.
 		maxNotFoundRetries := int32(utils.GetEnvInt("BCDA_WORKER_MAX_JOB_NOT_FOUND_RETRIES", 3))
 		if job.ErrorCount >= maxNotFoundRetries {
-			q.log.Errorf("No job found for ID: %d acoID: %s. Retries exhausted. Removing job from queue.", jobArgs.ID,
-				jobArgs.ACOID)
+			q.log.WithFields(ctxFields).Errorf("No job found. Retries exhausted. Removing job from queue.")
 			// By returning a nil error response, we're singaling to que-go to remove this job from the jobqueue.
 			return nil
 		}
 
-		q.log.Warnf("No job found for ID: %d acoID: %s. Will retry.", jobArgs.ID, jobArgs.ACOID)
+		q.log.WithFields(ctxFields).Warnf("No job found, retrying")
```

Comment:
like the log statements above and many of the following, I opted to remove values from the message that are now in their own field in the log message.

Reply:
Ah! I see, so this is also answers my other question regarding the interpolation that was removed.